hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java

```diff
@@ -176,7 +183,10 @@ public void updateTopicWithSchema(TopicName topicName, PatchData patch, String m
         Topic topic = getTopicDetails(topicName);
         boolean validateAvroSchema = AVRO.equals(topic.getContentType());
         extractSchema(patch)
-                .ifPresent(schema -> schemaService.registerSchema(topic, schema, validateAvroSchema));
+                .ifPresent(schema -> {
+                    schemaService.registerSchema(topic, schema, validateAvroSchema);
+                    scheduleTouchTopic(topicName);
```

Comment:
Why not touch always (even without schema being present), so this feature will be more consistent - "the topic is updated, the notification is sent"?

Reply:
In the below line we have `updateTopic()` which already sends notification when there are topic changes. However the problem was with only schema update. Previously Hermes didn't send any notification when only schema was updated so other instances were not aware of schema update. This is why I've added here topic touch.

Reply:
👍